pkgs/tools/system/gotop/default.nix

```diff
@@ -1,28 +0,0 @@
-{ stdenv, buildGoModule, fetchFromGitHub }:
-
-buildGoModule rec {
-  pname = "gotop";
-  version = "4.0.1";
-
-  src = fetchFromGitHub {
-    owner = "xxxserxxx";
```

Comment:
please revert this, `gotop` points to a maintained fork

Reply:
I should have checked first. Thanks